annotate.el

```diff
@@ -7,7 +7,7 @@
 ;; Maintainer: Bastian Bechtold
 ;; URL: https://github.com/bastibe/annotate.el
 ;; Created: 2015-06-10
-;; Version: 0.7.0
+;; Version: 0.7.1
```

Comment:
One minor nitpick: In the semantic versioning convention, we should bump the minor version, not the bugfix version. In short, the convention says that major version changes denote breaking changes, minor version changes are feature additions, and bugfix version changes only fix bugs.

Since this is a feature addition, it should result in 0.8.0.

(Although we might want to postpone that a bit if there are more things you'd like to add before pushing a new version)